about.html

```diff
@@ -0,0 +1,166 @@
+      <p class="about-past-details">Take a look our past Bootcamp students who successfully made their career.</p>
+      <ul class="past-item-group">
+        <li class="past-item">
+          <div class="student-img">
+            <img src="./images/raihan.jpg" alt="Abu Raihan">
+            <h5 class="student-bootcamp">Fullstack Bootcamp 2022</h5>
+          </div>
+          <div class="student-info">
+            <h3 class="sudent-name">Abu Raihan</h3>
+            <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+          </div>
+        </li>
+
+        <li class="past-item">
+          <div class="student-img">
+            <img src="./images/raihan.jpg" alt="Abu Raihan">
+            <h5 class="student-bootcamp">Fullstack Bootcamp 2021</h5>
+          </div>
+          <div class="student-info">
+            <h3 class="sudent-name">Abu Raihan</h3>
+            <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+          </div>
+          </li>
+
+          <li class="past-item">
+            <div class="student-img">
+              <img src="./images/raihan.jpg" alt="Abu Raihan">
+              <h5 class="student-bootcamp">Fullstack Bootcamp 2021</h5>
+            </div>
+            <div class="student-info">
+              <h3 class="sudent-name">Abu Raihan</h3>
+              <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+            </div>
+          </li>
+          
+          <li class="past-item">
+            <div class="student-img">
+              <img src="./images/raihan.jpg" alt="Abu Raihan">
+              <h5 class="student-bootcamp">Fullstack Bootcamp 2020</h5>
+            </div>
+            <div class="student-info">
+              <h3 class="sudent-name">Abu Raihan</h3>
+              <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+            </div>
+          </li>
+      </ul>
+    </section>
+
+  </main>
+  <footer class="footer black-footer">
+    <div class="footer-block">
+      <div class="logo">
+        <img src="./images/logo-grey.png" alt="Code with Raihan"/>
+      </div>
+      <h3>&#169;2022 Code with Raihan, Inc. All Rights Reserved.</h3>
+    </div>
+  </footer>
```

Comment:
- [ ] Please note that the footer background for the **_mobile version_** should be white both in the index and the about page, kindly fix it to match the design guidelines 🙏